Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
well, I don't really like pragmas in `.c` files unless absolutely necessary. So I'd move the init functions into a `pwm_init.c` module. Keeping the output functions in `pwm_output.c` seems sensible, since they are output functions.